Mathlib/Probability/Independence/Conditional.lean

```diff
@@ -796,11 +799,11 @@ section CommMonoid
 variable {m : MeasurableSpace β} [CommMonoid β] [MeasurableMul₂ β] {f : ι → Ω → β}
 
 @[to_additive]
-theorem iCondIndepFun.condIndepFun_finset_prod_of_not_mem
+theorem iCondIndepFun.condIndepFun_finset_prod_of_notMem
```

Comment:
looks like this one didn't get its deprecation (and also there should be a deprecation for the additive version)